recipes/fixie/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "fixie" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "49106b22f07b08a65473b8f851b9758547ac9a63742b1c5e1a4baa59e26acf2e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Will the `skip appveyor` be sufficient to make the `noarch: python` work correctly? Typically you can't have a `skip` in the `meta.yaml`. 

Otherwise the recipe looks good to me.

Reply:
Hi @synapticarbors - I have done this before and it has worked fine. Also note that there there is no skip in the meta.yaml

Reply:
Thanks!